app/lnd/config/index.js

```diff
@@ -2,8 +2,10 @@
 // Mac OS X: /Users/user/Library/Application Support/Lnd/tls.cert
 // Linux: ~/.lnd/tls.cert
 // Windows: TODO find out where cert is located for windows machine
+import { userInfo } from 'os'
+
 export default {
   lightningRpc: `${__dirname}/rpc.proto`,
   lightningHost: 'localhost:10009',
-  cert: '/Users/joaoalmeida/Library/Application\ Support/Lnd/tls.cert'
+  cert: '/Users/'+ userInfo().username+'/Library/Application\ Support/Lnd/tls.cert'
```

Comment:
again, super picky, but this should be \`/Users/${userInfo().username+}/Library/Application\ Support/Lnd/tls.cert\`

:)